docs/zh/reference/information_schema/task_runs.md

```diff
@@ -0,0 +1,24 @@
+---
+displayed_sidebar: "Chinese"
+---
+
+# task_runs
+
+`task_runs` 提供有关异步任务执行的信息。
+
+`task_runs` 提供以下字段：
+
+| 字段          | 描述                                                         |
+| ------------- | ------------------------------------------------------------ |
+| QUERY_ID      | 查询的 ID。                                                  |
+| TASK_NAME     | 任务的名称。                                                 |
+| CREATE_TIME   | 任务创建的时间。                                             |
+| FINISH_TIME   | 任务完成的时间。                                             |
+| STATE         | 任务的状态，包括 PENDING（待处理）、RUNNING（运行中）、FAILED（失败）和SUCCESS（成功）。 |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/27314732-3539-4b24-9458-9e00fddca682)
Wrap the values in code format.
Also add a space preceding SUCCESS.